README.md

```diff
@@ -38,10 +38,12 @@ full list of system requirements and installation instructions in the
 To check if FuseSoC is working, and to get an initial feeling for how FuseSoC
 works, you can try to simulate a simple hardware design from our core libray.
 
-First, create and enter an empty workspace
+The `fusesoc` install is specific to a project directory, and not a system-wide feature.
 
```

Comment:
I think we should use some different wording here. FuseSoC itself is system-wide. The build-directory is workspace (project) specific and the libraries containing core files and fusesoc.conf can be both system-wide or workspace-specific depeending on whether it was added with the `--global` flag